src/main/java/seedu/contax/ui/ResultDisplay.java

```diff
@@ -2,27 +2,44 @@
 
 import static java.util.Objects.requireNonNull;
 
+import java.util.List;
+
 import javafx.fxml.FXML;
-import javafx.scene.control.TextArea;
+import javafx.geometry.Insets;
 import javafx.scene.layout.Region;
+import javafx.scene.text.Text;
+import javafx.scene.text.TextFlow;
 
 /**
- * A ui for the status bar that is displayed at the header of the application.
+ * Ui for the status bar that is displayed at the header of the application.
  */
 public class ResultDisplay extends UiPart<Region> {
 
     private static final String FXML = "ResultDisplay.fxml";
 
     @FXML
-    private TextArea resultDisplay;
+    private TextFlow resultDisplay;
 
+    /**
+     * Constructor for ResultDisplay UI component
+     * Used to set initial properties of resultDisplay TextFlow
```

Comment:
I know this convention is not really followed by the original code, but try to use a verb for the first word. E.g. `Constructs`.

Reply:
thanks for the catch, resolved in https://github.com/AY2122S2-CS2103-W17-1/tp/pull/207/commits/5712c55b1563e8adfdcf92abef182a1197e32964